nx/include/switch/services/hid.h

```diff
@@ -310,7 +310,8 @@ typedef enum
     CONTROLLER_PLAYER_8 = 7,
     CONTROLLER_HANDHELD = 8,
     CONTROLLER_UNKNOWN  = 9,
-    CONTROLLER_P1_AUTO = 10, /// Not an actual HID-sysmodule ID. Only for hidKeys*(). Automatically uses CONTROLLER_PLAYER_1 when connected, otherwise uses CONTROLLER_HANDHELD.
+    CONTROLLER_P1_AUTO = 10, /// Not an actual HID-sysmodule ID. Only for hidKeys*(). Switches the commanding controller between handheld and player 1 based on activity.
```

Comment:
Is there any reason not to "just do":

CONTROLLER_PLAYER_1 = BIT(0),
CONTROLLER_PLAYER_2 = BIT(1),
...
CONTROLLER_P1_AUTO = CONTROLLER_HANDHELD | CONTROLLER_PLAYER_1,
CONTROLLER_ANY = 0xFFFFFFFF